src/main/java/uk/ac/ebi/eva/submission/service/SubmissionService.java

```diff
@@ -210,14 +210,20 @@ public boolean checkUserHasAccessToSubmission(SubmissionAccount account, String
         }
     }
 
-    public void sendMailNotificationForStatusUpdate(SubmissionAccount submissionAccount, String submissionId,
-                                                    SubmissionStatus submissionStatus, boolean success) {
+    public void sendMailNotificationToUserForStatusUpdate(SubmissionAccount submissionAccount, String submissionId,
+                                                          SubmissionStatus submissionStatus, boolean success) {
```

Comment:
Can we add the project title in the notification to user so that the user can recognise the project?

Reply:
added